draft-dekater-scion-dataplane.md

```diff
@@ -250,6 +254,24 @@ The full forwarding process for a packet transiting an intermediate AS consists
 **Note:** The current SCION implementation uses the UDP/IP protocol as underlay. However, the use of other underlay protocols is possible and allowed.
 
 
+### Configuration
+
+Border routers require mappings from SCION  interface IDs to underlay addresses. Such information must be supplied to each router in an out of band fashion (e.g in a configuration file). For each link to a neighbor, these values must be configured:
+
+- Local interface ID
+- Neighbor type (core, parent, child, peer), depending on link type (see [](#paths-links)). Link type depends on mutual agreements between the organizations operating the ASes at each end of each link.
+- Neighbor ISD-AS number
+- Neighbor interface underlay address
+
+In addition, each router must be configured with a list of interface IDs in use within the AS, containing:
+
+- Interface ID
+- "intra-protocol" address of the remote router
```

Comment:
Does that make sense? I tried to separate this in two lists, one for inter-AS neighbors and one for intra-AS SCION routers

Reply:
The phrase "intra-protocol" is completely obscure to me. We haven't used that previously, so we might want to make a longer, more explicit phrase. Otherwise, yeah that part probably deserves an explanation, but it might need to be longer.

Reply:
To be handled in #23